### PR TITLE
Added unit tests for files mixin

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
added unit tests for
- function `checkIfElementEixists` in `packages/web-app-files/src/mixins.js`
- `acceptShare.js` mixin

## Related Issue

- part of #5236

## How Has This Been Tested?

- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by saw-jan — COMMENTED** [review]
`acceptShare.js` seems to be inside folder `../src/mixins/actions`
so acceptShare spec file should be in `../unit/mixins/actions/acceptShare.spec.js`

**Comment by kiranparajuli589** [comment]
> `acceptShare.js` seems to be inside folder `../src/mixins/actions`
> so acceptShare spec file should be in `../unit/mixins/actions/acceptShare.spec.js`

fixed, please review again!

**Review by saw-jan — COMMENTED** [review]
LGTM

File: packages/web-app-files/tests/unit/mixins/actions/acceptShare.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  describe('computed property "$_acceptShare_items"', () => {
+    describe('isEnabled property of returned element', () => {
+      it.each([
+        { resource: { status: shareStatus.pending }, expectedStatus: true },
+        { resource: { status: shareStatus.declined }, expectedStatus: true },
+        { resource: { status: shareStatus.accepted }, expectedStatus: false }
+      ])(
+        'should be set w.r.t. the resource share status if the route name is "files-shared-with-me"',
```
Comment (individual-it) [inline packages/web-app-files/tests/unit/mixins/actions/acceptShare.spec.js:35]: what means `w.r.t`?

Comment (kiranparajuli589) [reply packages/web-app-files/tests/unit/mixins/actions/acceptShare.spec.js]: with respect to 😄 

> fixed

**Review by individual-it — APPROVED** [review]

**Pull request merged**